src/Forms/Main.cs

```diff
@@ -2736,13 +2736,24 @@ private void OpenSubtitle(string fileName, Encoding encoding, string videoFileNa
                     if (Configuration.Settings.SubtitleSettings.SamiDisplayTwoClassesAsTwoSubtitles && format.GetType() == typeof(Sami) && Sami.GetStylesFromHeader(_subtitle.Header).Count == 2)
                     {
                         var classes = Sami.GetStylesFromHeader(_subtitle.Header);
-                        var s1 = new Subtitle(_subtitle);

```

Comment:
We only need some properties, this will copy entire paragraph which are cleared in line 2740 and 27041.

Execution time optimized: plus 4n faster.